packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -152,18 +176,48 @@ export default decorate([
       </FormGrid.Row>
       {value.nbNodes > 0 &&
         value.staticIpAddress && [
-          <FormGrid.Row key='controlPlaneIpAddrRow'>
-            <label>{_('recipeControlPlaneIpAddress')}</label>
-            <input
-              className='form-control'
-              name='controlPlaneIpAddress'
-              onChange={effects.onChangeValue}
-              placeholder={formatMessage(messages.recipeControlPlaneIpAddress)}
-              required
-              type='text'
-              value={value.controlPlaneIpAddress}
-            />
-          </FormGrid.Row>,
+          value.highAvailability && [
+            [...Array(+3)].map((v, i) => (
+              <FormGrid.Row key={v}>
```

Comment:
Since the `.map` iterates over an array of `undefined` elements, `v` is `undefined`.

Reply:
You're right.

I know it's a bad practice to use the index as key. Do you have any advice in this situation?

Reply:
To make sure the key is unique, why not use the same value as the `name` attribute? `controlPlaneIpAddress.${i}`

Reply:
That's a good idea!